### DISCUSSION TITLE
Disable no-duplicate-imports in linter

### DISCUSSION THREAD

**Discussion by josejachuf** [quasarframework/quasar#14138] [General - Components / Directives / etc] [answered]
[quasar v1]

I need to import **L** this way

```
import { CRS, latLng, Icon } from 'leaflet' // line 186
import L from 'leaflet'                     // line 187

```
The explanation is here [1]

But the linter protests the double importation:

> 186:35  error  './node_modules/leaflet/dist/leaflet-src.js' imported multiple times  import/no-duplicates
> 187:15  error  './node_modules/leaflet/dist/leaflet-src.js' imported multiple times  import/no-duplicates

in eslinrc.js I add this line in rules:
`'no-duplicate-imports': 'off',`

But it has not worked

How do I avoid this error?

Jose

[1] https://github.com/vue-leaflet/Vue2Leaflet/issues/650

**Comment by josejachuf** [comment]
I think this rule works fine:

`'import/no-duplicates': 'off',`

**Comment by yusufkandemir** [comment, marked as answer]
You can disable the rule, but you are supposed to import it like this:
```js
import L, { CRS, latLng, Icon } from 'leaflet'
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#default_import
